jenkins_bcryptpw.py

```diff
@@ -30,12 +30,12 @@
     sys.stderr.write("USAGE: {} PASSWORD\n".format(basename(sys.argv[0])))
     sys.exit(_OS_EX_USAGE)
 
-plaintext = sys.argv[1]
+plaintext = sys.argv[1].encode('UTF-8')
 
 # https://gist.github.com/Voronenko/d50fc04cbbf26dfed37d
 # https://pypi.python.org/pypi/bcrypt/3.1.0
 pw = bcrypt.hashpw(plaintext, bcrypt.gensalt(10, prefix=b"2a"))
 
-print ("plaintext : {}".format(plaintext))
-print ("hash      : {}".format(pw))
-print ("check     : {}".format("passed" if bcrypt.checkpw(plaintext, pw) else "failed"))
```

Comment:
`checkpw(..)` is supported from `bcrypt` version [3.1.0](https://pypi.python.org/pypi/bcrypt/3.1.0). Can you check which version you have?

Reply:
ouch, Xenial 2.0.0

```
jrivero@nium ~ $ rmadison python3-bcrypt
 python3-bcrypt | 0.4-1ubuntu2  | trusty/universe | amd64, arm64, armhf, i386, powerpc, ppc64el
 python3-bcrypt | 2.0.0-2build1 | xenial/universe | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 python3-bcrypt | 3.1.3-1       | artful/universe | amd64, arm64, armhf, i386, ppc64el, s390x
 python3-bcrypt | 3.1.4-2       | bionic/universe | amd64, arm64, armhf, i386, ppc64el, s390x
```